### PR TITLE
MySQL Workshop 2 - Exercise 13

### PR DESCRIPTION
**Pull request by noratop**

### CODE REVIEW

File: exercise-13.txt [hunk 1]
```diff
@@ -0,0 +1,32 @@
+Exercise 13
+
+How many Address Book Entries have phones starting with area code 3XX?
+
+mysql> SELECT COUNT(*) as nbEntries FROM Entry WHERE contentLineOne LIKE '%(3__)%';
```
Comment (nyanofthemoon) [inline exercise-13.txt]: Would this also work for phone numbers with country codes like : 1-333-514-4560?

File: exercise-13.txt [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+| nbEntries |
++-----------+
+|        39 |
++-----------+
+1 row in set (0.00 sec)
+
+
+How many Address Book Entries have home phones not containing the digit 5?
+
+mysql> SELECT COUNT(*) AS nbEntries FROM Entry WHERE type='home' AND subtype='phone' AND contentLineOne NOT LIKE '%5%';
```
Comment (nyanofthemoon) [inline exercise-13.txt]: :+1:

File: exercise-13.txt [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+mysql> SELECT COUNT(*) AS nbEntries FROM Entry WHERE type='home' AND subtype='phone' AND contentLineOne NOT LIKE '%5%';              
++-----------+
+| nbEntries |
++-----------+
+|        76 |
++-----------+
+1 row in set (0.00 sec)
+
+
+How many Address Book Entries are work phones starting with a country code?
```
Comment (nyanofthemoon) [inline exercise-13.txt]: This would match (123) 543-6789 but not 1-123-543-6789. "1-" is the country code.

**Comment by noratop** [comment]
I should be a little tired because I am not able to read the instructions right!! What am I doing...I will review this exercise.

**Pull request closed without merging**